src/detect.h

```diff
@@ -515,6 +515,7 @@ typedef struct SignatureInitDataBuffer_ {
     bool sm_init; /**< initialized by sigmatch, which is likely something like `urilen:10; http.uri;
                      content:"abc";`. These need to be in the same list. Unset once `http.uri` is
                      set up. */
+    bool multi_capable; /**< true if we can have multiple instances */
```

Comment:
I think this could be more indicative of what multiple instances you have, do you mean multitenant? At least the comment could mention that.

Reply:
I see reference to "multi" multiple times. I think it might be a better idea to use e.g. mt_capable instead? with a comment explaining it is referring to multitenant?

Reply:
It's not related to multi-tenant, but to this ad88efc2d8688040360483101be3943973ac3691